### PR TITLE
Refactor MTE-2069 [v123] databaseFixturesTests with TR

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-2069)

## :bulb: Description
https://docs.google.com/document/d/1twjzxoAYf8ahWohifU5ZuwcHx0NNdAzkeItPBhPkOuU/edit
https://testrail.stage.mozaws.net/index.php?/suites/view/45443&group_by=cases:section_id&group_order=asc&group_id=471059

### CODE REVIEW

**Review by clarmso — APPROVED** [review]

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/DatabaseFixtureTest.swift [hunk 1]
```diff
@@ -23,18 +23,7 @@ class DatabaseFixtureTest: BaseTestCase {
         super.setUp()
     }
 
-    /* Disabled due to issue with db: 8281*/
-    /*func testOneBookmark() {
```
Comment (clarmso) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/DatabaseFixtureTest.swift:27]: Will we enable this test in the future?

Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/DatabaseFixtureTest.swift:27]: That particular scenario is covered in our bookmarks tests.

Comment (isabelrios) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/DatabaseFixtureTest.swift:27]: Can we remove it then? here or in a follow up PR...

Comment (clarmso) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/DatabaseFixtureTest.swift:27]: @dragosb01 Could you please ensure that this deleted test isn't referred in TestRail?

Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/DatabaseFixtureTest.swift:27]: This test has no reference in TR

**Pull request merged**